dashboard/lib/app.js

```diff
@@ -54,9 +59,13 @@ app.set('view engine', 'jade');
 //app.use(favicon(__dirname + '/public/favicon.ico'));
 
 app.use(stylus.middleware(
-    { src: __dirname + '/public', compile: compile
+    {
+        src: __dirname + '/stylus',
+        dest: __dirname + "/public/stylesheets",
+        compile: compile
     }
-));
+))
+;
 
 app.use(session({secret: 'ssshhhhh', title_timestamp: ''}));
```

Comment:
The secret is used to hash the session. Secret is required for use sessions. Probably we should change it periodically, but for now, the development for this value is TBD.